.changelog/release-v14.1.1.md

```diff
@@ -0,0 +1,125 @@
+15. event-sidecar: **v12.0.0**
+16. event-stream-processor: **v12.0.0-snapshot.7**
+17. simulator: **12.0.0**
+18. mojaloop-simulator: **v13.0.1**
+19. sdk-scheme-adapter: **v21.4.0**
+20. ml-testing-toolkit: v15.2.0 -> **v15.7.0** ([Compare](https://github.com/mojaloop/ml-testing-toolkit/compare/v15.2.0...v15.7.0))
+21. ml-testing-toolkit-ui: v15.0.1 -> **v15.1.3** ([Compare](https://github.com/mojaloop/ml-testing-toolkit-ui/compare/v15.0.1...v15.1.3))
+22. ml-testing-toolkit-client-lib: v1.0.0 -> **v1.0.1** ([Compare](https://github.com/mojaloop/ml-testing-toolkit-client-lib/compare/v1.0.0...v1.1.1))
+23. auth-service: **v13.0.2**
+24. als-consent-oracle: **v0.2.0**
+25. thirdparty-api-svc: **v13.0.2**
+26. thirdparty-sdk: **v15.1.0**
+
+## 4. Application release notes
+
+1. ml-api-adapter - https://github.com/mojaloop/ml-api-adapter/releases/tag/v14.0.0
+2. central-ledger - https://github.com/mojaloop/central-ledger/releases/tag/v16.3.1
+3. account-lookup-service - https://github.com/mojaloop/account-lookup-service/releases/tag/v14.0.0
+4. quoting-service - https://github.com/mojaloop/quoting-service/releases/tag/v15.0.2
+5. central-settlement- https://github.com/mojaloop/central-settlement/releases/tag/v15.0.0
+6. central-event-processor - https://github.com/mojaloop/central-event-processor/releases/tag/v12.0.0
+7. bulk-api-adapter - https://github.com/mojaloop/bulk-api-adapter/releases/tag/v14.2.0
+8. email-notifier - https://github.com/mojaloop/email-notifier/releases/tag/v12.0.0
+9. als-oracle-pathfinder - https://github.com/mojaloop/als-oracle-pathfinder/releases/tag/v12.0.0
+10. transaction-requests-service - https://github.com/mojaloop/transaction-requests-service/releases/tag/v14.0.1
+11. finance-portal-ui _(DEPRECATED)_ - https://github.com/mojaloop/finance-portal-ui/releases/tag/v10.4.3
+12. finance-portal-backend-service _(DEPRECATED)_ - https://github.com/mojaloop/finance-portal-backend-service/releases/tag/v15.0.2
+13. settlement-management _(DEPRECATED)_ - https://github.com/mojaloop/settlement-management/releases/tag/v11.0.0
+14. operator-settlement _(DEPRECATED)_ - https://github.com/mojaloop/operator-settlement/releases/tag/v11.0.0
+15. event-sidecar - https://github.com/mojaloop/event-sidecar/releases/tag/v12.0.0
+16. event-stream-processor - https://github.com/mojaloop/event-stream-processor/releases/v12.0.0-snapshot.7
+17. simulator - https://github.com/mojaloop/simulator/releases/tag/v12.0.0
+18. mojaloop-simulator - https://github.com/mojaloop/mojaloop-simulator/releases/tag/v13.0.1
+19. sdk-scheme-adapter - https://github.com/mojaloop/sdk-scheme-adapter/releases/tag/v21.4.0
+20. ml-testing-toolkit - https://github.com/mojaloop/ml-testing-toolkit/releases/tag/v15.7.0
+21. ml-testing-toolkit-ui - https://github.com/mojaloop/ml-testing-toolkit-ui/releases/tag/v15.1.3
+22. ml-testing-toolkit-client-lib - https://github.com/mojaloop/ml-testing-toolkit-client-lib/releases/tag/v1.1.0
```

Comment:
Same as above?
What version are we using here? :D

```suggestion
22. ml-testing-toolkit-client-lib - https://github.com/mojaloop/ml-testing-toolkit-client-lib/releases/tag/v1.1.1
```

Reply:
1.1.1 good catch.